src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,73 @@
+
 package core.basesyntax;
 
-public class WorkWithFile {
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
 
+public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] supplyBuy = calculateStatistic(fromFileName);
+        writeFile(toFileName, createReport(supplyBuy[0], supplyBuy[1]));
+    }
```

Comment:
```suggestion
        String[] dataFromFile = readFromFile(fromFileName);
        writeFile(toFileName, createReport(dataFromFile));
    }
```
lets make structure better